src/components/ImageGrid/ImageCell.vue

```diff
@@ -8,7 +8,7 @@
     <figure class="search-grid_item">
       <i :style="`padding-bottom:${iPadding}%`" />
       <NuxtLink
-        :to="localePath('/photos/' + image.id)"
+        :to="localePath('/image/' + image.id)"
```

Comment:
Thank you for changing /photos/ to /image/! It makes the terminology consistent.